### PR TITLE
Fix fs char

### PR DESCRIPTION
**Pull request by Daydreamer-ia**
Please do not create a Pull Request without creating an issue first.
#10067
## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by Daydreamer-ia** [comment]
Proposal: https://www.yuque.com/nacos/fh5oxd/gl7y6957xdnkqqmi

**Comment by Daydreamer-ia** [comment]
link to #10119

File: common/src/main/java/com/alibaba/nacos/common/pathencoder/PathEncoderManager.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+import java.nio.charset.Charset;
+import java.util.Collection;
+
+/**
+ * To expose interface from {@link PathEncoder}.
+ *
+ * @author daydreamer-ia
+ */
+public class PathEncoderManager {
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/pathencoder/PathEncoderManager.java:29]: Manager换成单例实现会不会比较好。

全静态方法感觉是个`Utils`

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/alibaba/nacos/common/pathencoder/PathEncoderManager.java (hunk 1 above)
Comment (Daydreamer-ia) [reply common/src/main/java/com/alibaba/nacos/common/pathencoder/PathEncoderManager.java:29]: done

**Pull request merged**